encord/ontology.py

```diff
@@ -85,3 +95,16 @@ def save(self) -> None:
 
     def _get_ontology(self):
         return self._querier.basic_getter(OrmOntology, self._ontology_instance.ontology_hash)
+
+    # def _get_skeleton_templates(self) -> Dict[str, SkeletonTemplate]:
```

Comment:
Old logic used to pull SkeletonTemplates down. Not strictly necessary if they are delivered with Ontology